src/Control/Algebra.hs

```diff
@@ -79,51 +82,55 @@ type Has eff sig m = (Members eff sig, Algebra sig m)
 
 -- | Construct a request for an effect to be interpreted by some handler later on.
 send :: (Member eff sig, Algebra sig m) => eff m a -> m a
-send = alg . inj
+send = alg id . inj
```

Comment:
`send` always operates in the topmost algebra, so we start by just passing `id`.